numba/core/byteflow.py

```diff
@@ -1088,6 +1182,35 @@ def _dup_topx(self, state, inst, count):
         for val in duped:
             state.push(val)
 
+    if PYVERSION in ((3, 12), ):
+        def op_CALL_INTRINSIC_1(self, state, inst):
+            # See https://github.com/python/cpython/blob/v3.12.0rc2/Include/
+            # internal/pycore_intrinsics.h#L3-L17C36
+            if inst.arg == 3:  # INTRINSIC_STOPITERATION_ERROR
+                state.append(inst, intrinsic="INTRINSIC_STOPITERATION_ERROR")
+                state.terminate()
+                return
+            if inst.arg == 5:  # UNARY_POSITIVE
+                val = state.pop()
+                res = state.make_temp()
+                state.append(inst, intrinsic="UNARY_POSITIVE",
+                             value=val, res=res)
+                state.push(res)
+                #self.op_UNARY_POSITIVE(state, inst)
```

Comment:
delete?

Reply:
Fixed in aacc674beda0817792f2d9296009474c583e186d